lib/puppet/provider/package/pkgin.rb

```diff
@@ -30,7 +30,7 @@ def self.prefetch(packages)
   end
 
   def self.instances
-    pkgin(:list).split("\n").map do |package|
+    pkgin("-p", :list).split("\n").map do |package|
```

Comment:
I just meant in terms of the provider being backward compatible with older versions of `pkgin` (i.e. can Puppet users use this pkgin provider with a < 0.7.0 pkgin?).  If that's not the case, then this should be fine.